pycon/templates/finaid/receipt_upload.html

```diff
@@ -0,0 +1,61 @@
+{% extends "site_base.html" %}
+
+{% load bootstrap_tags %}
+{% load compress %}
+{% load static %}
+{% load i18n %}
+{% load boxes_tags %}
```

Comment:
I don't think the compress, static, or boxes_tags template tag libraries are used in this template, so those loads can be omitted.